app/branding.js

```diff
@@ -307,7 +309,7 @@ export function getAssetNamespaces() {
  */
 export function getAssetHideNamespaces() {
     // e..g "OPEN.", "bit"
-    return [];
+    return ["ESCODEX."];
```

Comment:
This is not an acceptable change.

Reply:
I agree and I made this change after applying for mergers.
It shouldn't have come here.